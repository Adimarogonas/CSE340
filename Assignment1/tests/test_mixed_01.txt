
1 
1ZZ22.3B4  
IF WHILE * / $
