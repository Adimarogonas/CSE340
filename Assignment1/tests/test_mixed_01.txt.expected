{1 , NUM , 2}
{1 , NUM , 3}
{ZZ22 , ID , 3}
{ , DOT , 3}
{3 , NUM , 3}
{B4 , ID , 3}
{IF , IF , 4}
{WHILE , WHILE , 4}
{ , MULT , 4}
{ , DIV , 4}
{ , ERROR , 4}
{ , END_OF_FILE , 6}
